src/common/components/elements/AppHeader.tsx

```diff
@@ -91,11 +95,13 @@ export function AppHeader() {
               className={`min-w-fit`}
             >
               <Link href="/nfts/new" passHref>
-                <a className="hover:underline focus:underline">Mint NFTs</a>
+                <a className="text-lg font-medium text-gray-300 duration-100 ease-in hover:text-white focus:text-white">
+                  Create
+                </a>
               </Link>
             </HeaderLinkWrapper>
 
-            <Popover
+            {/* <Popover
```

Comment:
remove commented code unless this is a temporary change

Reply:
i kept it till we get the footer in